test/simpleTestForGenerator/ODbL-1.0.txt

```diff
@@ -1,15 +1,3 @@
-ODC Open Database License (ODbL)
-
-Preamble
```

Comment:
How did you generate this new content?  When I run:

```console
$ curl -s https://opendatacommons.org/files/2018/02/odbl-10.txt | dos2unix >test/simpleTestForGenerator/ODbL-1.0.txt
```

I still get the preamble.

Reply:
I cut out the preamble.  After this is merged, I can create a separate PR with the preamble.

I don't want to confuse this PR too much by adding in optional text for the preamble in addition to the more important fix of the section formatting.  

I included the text since I needed to update the test file anyway and thought I should just go with the original source minus the preamble.

Reply:
When I copy the text verbatim, the match fails.  I'll need to investigate why.